schema/omega-services.async.api.yaml

```diff
@@ -59,7 +60,7 @@ channels:
           - $ref: '#/components/messages/editSet'
           - $ref: '#/components/messages/editSetSummary'
           - $ref: '#/components/messages/editSetRecord'
-          - $ref: '#/components/messages/updateEditSetRecord'
+          - $ref: '#/components/messages/editSetRecordUpdate'
```

Comment:
We can remove this from here. When the user clicks "Save changes" they are always taken to the confirmation page and don't get the whole record back unless they return to the edit record page, in which case they would make a fresh call to `getEditSetRecord` .

Reply:
@rwalpole in that case, the `editSetRecordUpdate` message seems redundant as this is not being used and can be removed too?

Reply:
> We can remove this from here. When the user clicks "Save changes" they are always taken to the confirmation page and don't get the whole record back unless they return to the edit record page, in which case they would make a fresh call to `getEditSetRecord` .

Removed.

Reply:
Agreed @userjd . I think there are some other inconsistencies in here as well which I have raised in https://national-archives.atlassian.net/browse/PACT-789.